src/cap-description.adoc

```diff
@@ -525,9 +525,10 @@ malformed    =  !EF && (malformedMSB || malformedLSB)
 NOTE: The check is for malformed _bounds_, so it does not include reserved
 bits!
 
-It is impossible for a CHERI core to generate a tagged capability with malformed
-bounds, or with any reserved bits set. If such a capability exists then it must
```

Comment:
I would keep the "If such a capability exists then it must have been caused by a logic or memory fault." sentence (rephrased to match the new wording).

Reply:
I pushed changes to add this back